decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/moderations_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
```

Comment:
In a later commit you are moving categories & moderations to the plugin, remember to move them back to core :)